packages/angular_devkit/build_angular/test/browser/rebuild_spec_large.ts

```diff
@@ -477,4 +478,134 @@ describe('Browser Builder rebuilds', () => {
       .toPromise();
     await run.stop();
   });
+
+  it('rebuilds AOT works with library HTML and CSS changes', async () => {
+    // DISABLED_FOR_IVY - Not yet supported under Ivy
+    if (ivyEnabled) {
+      pending('Not supported in Ivy');
```

Comment:
Is this not supported or just not relevant? I imagine it should just work. If it does, we should test it.

Reply:
It’s not supported because ngcc is non incremental. This should eventually work when we have dev libs in ivy mode.